src/iproject.ts

```diff
@@ -68,39 +106,127 @@ export class IProject {
     return Uri.file(path.join(this.workspaceFolder.uri.fsPath, `.env`));
   }
 
-  public async read() {
-    const content = await workspace.fs.readFile(this.getIProjFilePath());
-    this.state = IProject.validateIProject(content.toString());
+  public async addToIncludePaths(includePath: string) {
+    const unresolvedState = await this.getUnresolvedState();
+
+    if (unresolvedState) {
+      if (unresolvedState.includePath) {
+        if (!unresolvedState.includePath.includes(includePath)) {
```

Comment:
Do we need to try to make includeDirs relative to the project before adding them to the path?

Reply:
@worksofliam I think we discussed this earlier? I can't remember exactly, but was there a reason we didn't want to do this or can I go ahead and make the change to relative includeDirs?

Reply:
#21